syntax = "proto3";

package Univer;

message FullName {
    string name = 1;
    string surname = 2;
    optional string patronymic = 3;
}

message Student {
    FullName name = 1;
    repeated int32 scores = 2;
    int32 sc_average = 3;
}

message StudentsGroup {
    repeated Student students = 1;
}